src/codegen_new/codegen_backend_x86-64_ops.c

```diff
@@ -68,8 +68,10 @@ jmp(codeblock_t *block, uintptr_t func)
 void
 host_x86_ADD8_REG_IMM(codeblock_t *block, int dst_reg, uint8_t imm_data)
 {
+#if 0
```

Comment:
Just a thought: why not use some specific defined name in all of these ifdefs ?

This would make it easier to switch between the enabled and disabled states.

Reply:
Done.